table_loading.py

```diff
@@ -391,27 +391,36 @@ def fit_a_source(data, error, valid, geometry='s-ubhmi',
 magcols = ['Ymag', 'Zmag', 'Jmag', 'Hmag', 'Ksmag','mag3_6', 'mag4_5', 'mag5_8', 'mag8_0']
 emagcols = ['Yell', 'Zell', 'Jell', 'Hell', 'KsEll','e_mag3_6', 'e_mag4_5', 'e_mag5_8', 'e_mag8_0']
 
-
 def mag_to_flux(tbl, magcols, emagcols, zpts, filternames):
- 
-    # convert magnitudes to fluxes now
+    # convert magnitudes to fluxes
     # (it's a pain to try to deal with a mix of magnitudes & fluxes)
     for colname, errcolname, zpn in zip(magcols, emagcols, filternames):
-        print(colname, zpn)
         zp = u.Quantity(zpts[zpn], u.Jy)
-        if colname in tbl.keys():
-            data = tbl[colname].value
+        # iterate through each colname
+        if colname and errcolname in tbl.keys():
```

Comment:
do you mean 
```suggestion
        if colname in tbl.keys() and errcolname in tbl.keys():
```
?  The current version will be True always, unless `colname` is False